.gitlab-ci.yml

```diff
@@ -22,22 +19,18 @@ variables:
 .build:                            &docker_build
   stage:                           build
   image:                           docker:stable
-  only:
-    refs:
```

Comment:
@gabreal FYI, I'm removing this since CI is triggered only by $DOCKERFILES variable anyway